uber/site_sections/preregistration.py

```diff
@@ -150,8 +152,9 @@ def form(self, session, message='', edit_id=None, **params):
             return render('static_views/dealer_reg_closed.html') if c.AFTER_DEALER_REG_SHUTDOWN else render('static_views/dealer_reg_not_open.html')
 
         if 'first_name' in params:
-            message = check(attendee) or check_prereg_reqs(attendee)
+            message = check(attendee) or check_prereg_reqs(attendee, group)
             if not message and attendee.badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE]:
+                check_prereg_reqs(attendee, group)
```

Comment:
Calling this function here standalone does nothing.  I assume this was included by mistake?

Otherwise great work and +1 on merging this after this superfluous call is removed.